cypress/integration/publishingFlows/mentionUser.js

```diff
@@ -0,0 +1,98 @@
+describe('Article Editor', () => {
```

Comment:
nitpick (non-blocking): We already have a describe block in the other editor tests file called `'Article Editor'`. Consider calling this one `'Article Editor (Mention User)'` or something along those lines. An alternative is to include these files in the existing editor specs file.,